packages/api/src/auth/verifiers/jwtVerifier.ts

```diff
@@ -0,0 +1,90 @@
+import jwt from 'jsonwebtoken'
+
+import {
+  VerifyOptions,
+  WebhookSignError,
+  WebhookVerificationError,
+  DEFAULT_WEBHOOK_SECRET,
+} from './index'
+import type { WebhookVerifier } from './index'
+
+export type JwtVerifier = WebhookVerifier
```

Comment:
Another circular type I think. We can probably not export this here

Reply:
@dac09 Can we chat in person about this ^